src/transformers/models/bert/modeling_bert.py

```diff
@@ -1153,7 +1155,7 @@ def forward(
 class BertLMHeadModel(BertPreTrainedModel):
 
     _keys_to_ignore_on_load_unexpected = [r"pooler"]
-    _keys_to_ignore_on_load_missing = [r"position_ids", r"predictions.decoder.bias"]
+    _keys_to_ignore_on_load_missing = [r"position_ids", r"predictions.decoder.bias", r"cls.predictions.decoder.weight"]
```

Comment:
A re match is done, so while the exact name is indeed `r"cls.predictions.decoder.bias"`, this works. But would be great to fix just in case one day a weight named `predictions.decoder.bias` that should not be ignore appears ;-)